ts/components/wallet/WalletLayout.tsx

```diff
@@ -107,8 +112,10 @@ export class WalletLayout extends React.Component<Props> {
           </View>
         );
       }
+      default:
+        ((): never => this.props.cardType)();
```

Comment:
is this construction to avoid tsc complaining about unused var?

Reply:
Exactly, I figured it has the same purpose as what you proposed, but without the warning :)